src/azure-cli/azure/cli/command_modules/policyinsights/tests/latest/test_policyinsights_scenario.py

```diff
@@ -364,7 +364,7 @@ def test_policy_insights_remediation_management_group(self):
     @AllowLargeResponse()
     def test_policy_insights_remediation_complete(self):
         self.kwargs.update({
-            'pan': '09d18943ace14946aff83c21',
+            'pan': '98904c39668a4f70804aef09',
```

Comment:
May I ask why you need to change this value?

Reply:
This test test_policy_insights_remediation_complete also requires test environment setup. There are several steps in the comments in the front of the test. Updating the policy assignment value is in the step 4. (The root reason is when we create the assignment on policy UI, the resource name is generated by the UI. We cannot specify the assignment name on the UI)

Reply:
Do you mean that the steps to set up the test environment must be on portal?
If so, why can't it be implemented by commands in the test?

Reply:
Yes, it needs to setup through Portal. I don't know why wasn't it implemented by cmd before. I guess the scenario depends on many async operations and needs a long wait time for backend job could be potentially the blocker

Reply:
Actually, it described the reason in the comment above: 
```
    # Executing a real remediation requires more time-intensive setup than can be done in a live scenario test.
    # This record_only test executes a real remediation against a known non-compliant policy
```

Reply:
I suggest trying to use commands to set up the environment, otherwise it is not easy to maintain and reuse this test in the future

Reply:
The test pre-requirements of this test needs to create other resources like VM. If we creates those resources on the fly, the test will become the same situation as that we are hitting right now on the SQL test. I think ideally, the command module should not create dependencies on other command modules. E.g. the current broken SQL test is not do any validation on the policy CLI cmd. They just called policy CLI cmd to update their test env to unblock the SQL managed instance creation. I think that is not proper. I think SDK team should discuss about the strategy/restriction of calling other command module in the command module scenario test. The product team may not response to the re-record request in time. It will become un-necessary blocker.

Reply:
At present, the tests of some modules in CLI depend on the commands of other modules to construct the environment, so it is quite common to call other module commands in testing.
It is a known issue that bumping version for some RP will cause other module tests which rely on this RP to need to be re-recorded.
For the creation of some resources, we can use prepare (such as @ResourceGroupPreparer,@StorageAccountPreparer) to avoid this problem, but VM is not supported at present.
If it's not easy to build a test environment with commands for this case, we can pass this problem first.

Reply:
Let's pass this problem first.